help/en/html/tools/scripting/Start.shtml

```diff
@@ -280,10 +280,26 @@ You can also use the symbols "True" and "False" respectively.
       send a DCC packet to the rails you type:</p>
       <pre>
    
-   dcc.sendPacket([0x01, 0x03, 0xbb], 4) 
+   <code>dcc.sendPacket([0x01, 0x03, 0xbb], 4) </code>
 </pre>This sends that three-byte packet four times, and then
 returns to the command line.
 
+
+    <p>If the returned object is an ID Tag or RailCom Object, the return value 
+    ( from Java #toString() ) is the Hardware Address.<br>
+    
+    If you require the ID Tag UserName, go via the Reportable Interface.<br>
```

Comment:
If you want the user name of any named bean, you ask for the user name of that named bean using the getUserName method.  

The toReportString provides an object specific report, which my or may not actually contain the user name by itself.  

LocoNet TranspondingTag report strings such as “1234 enter” and “1234 exit”.  There are a couple of other variations as well.

  Railcom objects will add any detector supplied values, such as orientation, to the string, something like “1234 orientation A” is a valid report.

( I am writing the report strings from memory, so the strings above may not be 100% correct, but they have the right behavior. )